QSPIFBlockDevice.cpp

```diff
@@ -827,79 +826,76 @@ int QSPIFBlockDevice::_sfdp_set_quad_enabled(uint8_t *basic_param_table_ptr)
 
     switch (qer_value) {
         case 0:
-            tr_debug("DEBUG: Device Does not Have a QE Bit, continue based on Read Inst");
+            tr_debug("Device Does not Have a QE Bit, continue based on Read Inst");
             return 0;
 
         case 1:
         case 4:
-            status_reg_setup[0] = 0;
-            status_reg_setup[1] = 0x02;  //Bit 1 of Status Reg 2
-            tr_debug("DEBUG: Setting QE Bit, Bit 1 of Status Reg 2");
+            status_reg_setup[1] = (0x1 << 1);  //Bit 1 of Status Reg 2
+            tr_debug("Setting QE Bit, Bit 1 of Status Reg 2");
             break;
 
         case 2:
-            status_reg_setup[0] = 0x40; // Bit 6 of Status Reg 1
-            status_reg_setup[1] = 0;
+            status_reg_setup[0] = (0x1 << 6); // Bit 6 of Status Reg 1
             sr_write_size = 1;
-            sr_read_size = 1;
-            tr_debug("DEBUG: Setting QE Bit, Bit 6 of Status Reg 1");
+            tr_debug("Setting QE Bit, Bit 6 of Status Reg 1");
             break;
 
         case 3:
-            status_reg_setup[0] = 0x80; // Bit 7 of Status Reg 1
-            status_reg_setup[1] = 0;
+            status_reg_setup[0] = (0x1 << 7); // Bit 7 of Status Reg 1
             sr_write_size = 1;
-            sr_read_size = 1;
             write_register_inst = 0x3E;
             read_register_inst = 0x3F;
-            tr_debug("DEBUG: Setting QE Bit, Bit 7 of Status Reg 1");
+            tr_debug("Setting QE Bit, Bit 7 of Status Reg 1");
             break;
         case 5:
-            status_reg_setup[0] = 0;
-            status_reg_setup[1] = 0x02; //Bit 1 of status Reg 2
+            status_reg_setup[1] = (0x1 << 1); // Bit 1 of status Reg 2
             read_register_inst = 0x35;
             sr_read_size = 1;
-            tr_debug("DEBUG: Setting QE Bit, Bit 1 of Status Reg 2 -special read command");
+            tr_debug("Setting QE Bit, Bit 1 of Status Reg 2 -special read command");
             break;
         default:
-            tr_warning("WARNING: _setQuadEnable - Unsuported QER configuration");
+            tr_warning("_setQuadEnable - Unsuported QER configuration");
             break;
-
-
     }
 
+    // Configure  BUS Mode to 1_1_1 for all commands other than Read
+    _qspi_configure_format(QSPI_CFG_BUS_SINGLE, QSPI_CFG_BUS_SINGLE, QSPI_CFG_ADDR_SIZE_24, QSPI_CFG_BUS_SINGLE,
```

Comment:
Is this the missing bus configuration that caused the problem with L475 ?

Reply:
Yes this was the fix, explained in jira and missed to add comment here

Reply:
Thanks!